src/main/java/io/jenkins/update_center/IndexHtmlBuilder.java

```diff
@@ -34,8 +37,17 @@
 import java.io.PrintWriter;
 import java.io.File;
 import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
There's a bunch of unused imports in this file.

Reply:
Imports fixed. Also now it's using the template from jenkins.io so the code could be simplified a bit.

Reply:
> should I merge master to this or rebase?

Either works, I'd probably squash anyway. Thanks for checking!